core/templates/dev/head/editor/StateResponses.js

```diff
@@ -306,6 +306,34 @@ oppia.controller('StateResponses', [
       trainingDataService.initializeTrainingData(
         explorationId, currentStateName);
     };
+    $scope.suppressDefaultAnswerGroupWarnings = function() {
+      var interactionId = $scope.getCurrentInteractionId();
+      if (interactionId === 'MultipleChoiceInput') {
+        var choices = $scope.getAnswerChoices().length;
```

Comment:
Yeah it works great!
Done
So the idea I should be trying to emulate is making variable names as explicit as possible about their purpose?